cmds/mount/mount.go

```diff
@@ -25,14 +27,48 @@ var (
 	opt    = flag.String("o", "", "Specify mount options")
 )
 
+// extended from boot.go
+func getSupportedFilesystem(originFS string) ([]string, bool, error) {
+	var known bool
+	var err error
+	fs, err := ioutil.ReadFile("/proc/filesystems")
+	if err != nil {
+		return nil, known, err
+	}
+	var returnValue []string
+	for _, f := range strings.Split(string(fs), "\n") {
+		n := strings.Fields(f)
+		if len(n) != 1 {
+			continue
+		}
+		if n[0] == originFS {
+			known = true
+		}
+		returnValue = append(returnValue, n[0])
+	}
+	return returnValue, known, err
+
+}
+
+func informIfUnknownFS(originFS string) {
+	knownFS, known, err := getSupportedFilesystem(originFS)
+	if err != nil {
+		// just don't make things even worse...
+		return
+	}
+	if !known {
+		log.Printf("Hint: unknown filesystem %s. Known are: %v\n", originFS, knownFS)
```

Comment:
log already adds new lines - remove the \n?

Reply:
will do.